src/features/user/ManageProjects/components/ProjectCertificates.tsx

```diff
@@ -238,7 +230,10 @@ function ProjectCertificates({
                                 </div> */}
                 <button
                   id={'trashIconProjC'}
-                  onClick={() => deleteProjectCertificate(report.id)}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    deleteProjectCertificate(report.id);
```

Comment:
Why was this needed?

Reply:
The button was inside a form, which caused unexpected navigation on the onClick event. Initially, I used preventDefault() to prevent this behavior, but I have now removed it by explicitly setting `type="button"` on the button.